server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -63,6 +77,8 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
       );
     }
 
+    const foreignKeyToColumnName = `${camelCase(record.toName)}Id`;
```

Comment:
Why this new naming? With the "to" it sounds like you are converting the fk to a column name but here you are using the field name right?

Reply:
@Weiko You're right, it was because during my test at a moment I had generate the foreign key on both sides. But I remove that part that was no needed.
I rollback the name ! :)